src/cryptoadvance/specter/templates/includes/address-data.html

```diff
@@ -108,6 +110,16 @@ <h1>{{ _("Address details") }}</h1><br>
                     let walletName = jsonResponse.walletName;
                     let address = jsonResponse.address;
                     let walletLink = `{{ url_for('wallets_endpoint.wallet', wallet_alias='WALLET_ALIAS') }}`.replace("WALLET_ALIAS", this.wallet);
+
+                    var formattedAmountAndUnit = formatBtcAmountAndUnit(this.amount);
+                    {% if specter.is_liquid %}
+                        var labels = [];
```

Comment:
code in html template - is it possible to extract to helper?

Reply:
Unfortunately the structure of `this.assets` and `this.tx` in https://github.com/relativisticelectron/specter-desktop/blob/677c971bf70393f064ac048bd8970bbea1131c56/src/cryptoadvance/specter/templates/includes/tx-row.html#L254 is different. I chose therefore a generic approach: Do not assume a specific format and simply pass arrays (or single values).